polygeist/tools/cgeist/Test/Verification/sycl/single_task.cpp

```diff
@@ -25,31 +21,20 @@
 // Test that all referenced sycl header functions are generated.
```

Comment:
Is this comment accurate? It seems to test that *no* functions are generated.

Reply:
It is accurate. We want to check that the functions are `define` and not `declare`.

Reply:
Ok, makes sense, thanks for the explanation!